snapcraft.yaml.in

```diff
@@ -0,0 +1,50 @@
+name: gaia # you probably want to 'snapcraft register <name>'
+# base: core18 # the base snap is the execution environment for this snap
+version: '@VERSION@' # just for humans, typically '1.2+git' or '1.3.2'
+summary: Gaia Daemon # 79 char long summary
+description: |
+  This snap provides the Gaia daemon gaiad and the command line
+  tool gaiacli.
+grade: devel # must be 'stable' to release into candidate/stable channels
+confinement: strict # use 'strict' once you have the right plugs and slots
+
+apps:
+    gaiad:
+        command: bin/gaiad
+        plugs: [home,network,network-bind]
+    gaiacli:
+        command: bin/gaiacli
+        plugs: [home,network,network-bind]
+
+parts:
+  gaia:
+    plugin: dump
+    source: ./
+    override-pull: |
+        rootdir=$(pwd)
+        gitroot=$(git rev-parse --show-toplevel)
+        cd ${gitroot} && git archive \
+            -o ${rootdir}/gaia-@VERSION@.tar.gz \
+            --format tar.gz -9 --prefix gaia-@VERSION@/ HEAD
+        cd ${rootdir}
+        tar xvf gaia-@VERSION@.tar.gz ; rm -f gaia-@VERSION@.tar.gz
+        mkdir -p go/src/github.com/cosmos bin
+        mv gaia-@VERSION@/ go/src/github.com/cosmos/cosmos-sdk/
+
+        # Use the following instructions to build a package from a release.
+        # wget https://github.com/cosmos/cosmos-sdk/archive/v0.29.0.tar.gz
+        # tar xvf v0.29.0.tar.gz
+        # rm v0.29.0.tar.gz
+
+    build-snaps: [go]
+    override-build: |
+        base=`pwd`
+        export GOPATH=`pwd`/go
```

Comment:
Is that always the case?

Reply:
Yes, at this stage it is in the sources root of the gaia `part`.